src/main/java/org/jenkinsci/plugins/runselector/selectors/SimpleRunSelectorDescriptor.java

```diff
@@ -1,67 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2004-2010, Sun Microsystems, Inc., Alan Harder
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package org.jenkinsci.plugins.runselector.selectors;
-
-import org.jenkinsci.plugins.runselector.RunSelector;
-import org.jenkinsci.plugins.runselector.RunSelectorDescriptor;
-import org.jvnet.localizer.Localizable;
-
-/**
- * Descriptor type for common case where no overrides are needed.
- * Just do: {@code @Extension public static final Descriptor<RunSelector> DESCRIPTOR =
- *          new SimpleRunSelectorDescriptor(MySelector.class, Messages._My_DisplayName()); }
- * @author Alan Harder
- */
-public class SimpleRunSelectorDescriptor extends RunSelectorDescriptor {
```

Comment:
I vote +1 for removing this.
I don't think this is useful. Plugins should implement `doCheckFoo` or `doFillBarItems` and this utility class doesn't encourage that.